lib/api.js

```diff
@@ -53,7 +54,44 @@ function get(config, uri, done) {
       if (!res.body) {
         return done(undefined, res.body, res);
       }
-      done(null, res.body, res);
+
+      var results = res.body;
+
+      if (res.headers.link) {
+        var numberOfPages = parseLinkHeader(res.headers.link).last.page;
+        var i = 1;
+
+        async.whilst(
+          function() {
+            return i < numberOfPages;
+          },
+          function(callback) {
+            superagent
+              .get(url)
+              .query({
+                private_token: config.api_key,
+                per_page: 100,
+                page: i
+              })
+              .set('User-Agent', 'StriderCD (http://stridercd.com)')
+              .end(function(err, res) {
+                if (err) return callback(err);
+
+                results = results.concat(res.body);
+                callback(null, i);
+              });
+
+            i++;
+            },
+            function(err, n) {
+              if(err) return done(err);
+
+              return done(null, results, res);
+            }
+          );
```

Comment:
85-91 should probably be un-indented one time, otherwise everything looks good :+1: